osbuild/util/chroot.py

```diff
@@ -0,0 +1,41 @@
+import os
+import subprocess
+
+
+class Chroot:
```

Comment:
Same comment as in https://github.com/osbuild/osbuild/pull/1846#discussion_r1718051103:
```
(super nitpick) it took me a moment to realize that this sets up the dirs but not doing the chroot()ing so I wonder if a slightly better name might be DracutMounts or MountedDevProcSys or something
```

Reply:
Yeah, I kept the name and added the note that it might be nice to have a `run()` method on the class because I thought that'd be nicer, but left it for a follow-up.  Probably best to just name it properly though.

Reply:
How's that?

Reply:
Yeah, let's rename in a followup, I really like the "call()" idea :)